learn/what_is_meilisearch/telemetry.md

```diff
@@ -145,6 +145,11 @@ This list is liable to change with every new version of Meilisearch. It's not be
 | `q.max_terms_number`                    | Highest number of terms given for the `q` parameter                                         | 5
 | `pagination.max_limit`                  | Highest value given for the `limit` parameter                                               | 60
 | `pagination.max_offset`                 | Highest value given for the `offset` parameter                                              | 1000
+| `formatting.highlight_pre_tag`          | `true` if `highlightPreTag` was used in this batch, otherwise `false`                       | false
```

Comment:
>`true` if `highlightPreTag` was used in this batch, otherwise `false` 

What do we mean by "this batch" here?

Reply:
As all search queries are grouped into a single search event per hour for analytics, we will have the information at `true` or `false` if it occurred at least once among all search queries. This is what batch is referring to here.

Reply:
We have been leaving the concept of `batches` out of the telemetry table on the docs since it's described in the [intro paragraph](https://docs.meilisearch.com/learn/what_is_meilisearch/telemetry.html#exhaustive-list-of-all-collected-data) and applies to every telemetry metric except perhaps version.

```suggestion
| `formatting.highlight_pre_tag`          | `true` if `highlightPreTag` was specified, otherwise `false`                       | false
```

Would recommend making equivalent changes to the other new table entries.